src/class.cscf_contact.php

```diff
@@ -46,6 +47,11 @@ function __construct() {
 						case 'message':
 							$this->Message = sanitize_textarea_field( $value );
 							break;
+						case 'phone-number':
+							if ( cscf_PluginSettings::PhoneNumber() ) {
+								$this->PhoneNumber = sanitize_text_field( $value );
+							}
+							break;
 						case 'contact-consent':
 							if ( cscf_PluginSettings::ContactConsent() ) {
```

Comment:
Same reason, just sanitizing so no reason to add if as it wont get here if it isn't

Reply:
Yep, I agree.